stories/topics/proc-aws-generate-remove-data-files.adoc

```diff
@@ -13,6 +13,13 @@ These commands create a directory, and  an empty data template that, when popula
 $ mkdir command_generator_data
 ----
 . Populate the `$(pwd)/command_generator_data` folder with the configuration file template.
++
+
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:tech-note-linux-files-owned-by-root[tech notes] for more information.
```

Comment:
generator is owned by....

For more information, read the xref:tech-note-linux-files-owned-by-root[tech notes].